### ISSUE TITLE
light opacity and full block fields do nothing.

### ISSUE THREAD

**Issue by Rosethorns** [The-Acronym-Coders/ContentTweaker#82]
Minecraft: 1.12.2
Forge: 14.23.2.2613
Content Tweaker: 4.4.1

Between the wiki and the project page there are two styles these can be called. So for the sake of completeness I have covered both

Checking with both of these produces the same results. Pictured is the slab block with fullBlock set to false. true (the default) produces the same result
block.fullBlock = false;
block.setFullBlock(false);
![image](https://user-images.githubusercontent.com/2171914/36352760-db60ce06-1522-11e8-959e-e42b642c959a.png)

Checking with both of these produces the same results. Pictured is the slab block with lightOpacity set to 0. 255 (the default) produces the same result
block.lightOpacity = 0;
block.setLightOpacity(0);
![image](https://user-images.githubusercontent.com/2171914/36352777-f04a6980-1522-11e8-9d8b-b423ff59068b.png)

This picture may be harder to tell as it is a dark block but the center-ish area of the 4 "slabs" (fake) is darker

Edit: Log says nothing about either style of using the field being invalid. No error.

Edit 2: with direction from Jared I also tried block.translucent = true;

so to recap.
currently the block has
block.lightOpacity = 0;
block.fullBlock = false;
block.translucent = true;
No log error.
Result
![image](https://user-images.githubusercontent.com/2171914/36353043-1ffe3fa0-1526-11e8-8476-d05e141da58c.png)

**Comment by SkySom** [comment]
var slab = VanillaFactory.createBlock("slab", <blockmaterial:cake>);
slab.fullBlock = false;
slab.lightOpacity = 0;
slab.translucent = true;
slab.register();

This works

**Comment by Rosethorns** [comment]
you mean it works after a bug fix or it always worked? cos I tried that series of settings. its literally the last one mentioned :P

**Comment by SkySom** [comment]
Yeah totally wasn't clear. 4.5 fixes it and now it works!
